src/query/v2/plan/operator.cpp

```diff
@@ -78,12 +78,16 @@ extern const Event OnceOperator;
 extern const Event CreateNodeOperator;
 extern const Event CreateExpandOperator;
 extern const Event ScanAllOperator;
+extern const Event ScanAllOperator_Distributed;
```

Comment:
Quick question, why with `_` (`_Distributed`)?

Reply:
It's temporary, the PR is still as a draft for now